cypress/e2e/tests/pages/charts/istio.spec.ts

```diff
@@ -8,7 +8,7 @@ describe('Charts', { tags: ['@charts', '@adminUser'] }, () => {
   const chartsPageUrl = '/c/local/apps/charts/chart?repo-type=cluster&repo=rancher-charts';
 
   describe('Istio', () => {
-    const istioVersion = '102.2.0%2Bup1.17.2';
+    const istioVersion = '104.0.0%2Bup1.18.2';
 
     const chartsIstioPage = `${ chartsPageUrl }&chart=rancher-istio&version=${ istioVersion }`;
```

Comment:
Do we need to include a chart version? I did one of these tests and I'm sure we no version, it will go to the latest version?

Reply:
i made a suggestion somewhere in slack to navigate to the istio chart page first and install from there, that will pick up whatever the latest version is